projects/isy-angular-widgets/src/lib/hauptfenster/hauptfenster.component.html

```diff
@@ -83,15 +84,16 @@
             (click)="collapsedLinksnavigation = false"
           ></button>
         </div>
-      </div>
+      </aside>
     }
 
-    <div class="col overflow-auto">
+    <div role="main" aria-label="main" class="col overflow-auto">
       <ng-content></ng-content>
     </div>
 
     @if (showInformationsbereich) {
-      <div
+      <aside
+        aria-label="informationsbereich"
```

Comment:
Translation should be used here. As otherwise screen readers only read out in German